src/components/app.jsx

```diff
@@ -80,23 +70,49 @@ const App = React.createClass({
                 <div className="col-md-9">
                   <p id="baseline">
                     <span className="message">
-                      <FormattedMessage {...messages.header} />
+                      <FormattedMessage id = "header"/>
+                      <small>
+                        <ExternalLink href = "https://www.openfisca.fr" target = "_blank">
+                          <strong><FormattedMessage id = "learnMore"/></strong>
```

Comment:
Was `<span class="variableMetadataHighlight">` better than `<strong>` ? If so, more explanations would be welcome.